apis/src/pages/tournament.rs

```diff
@@ -253,13 +253,11 @@ fn LoadedTournament(tournament: Signal<TournamentResponse>) -> impl IntoView {
     };
 
     view! {
-        <div class="flex flex-col justify-center p-2 w-full">
+        <div class="flex flex-col items-center p-2 w-full">
             <h1 class="w-full max-w-full text-3xl font-bold text-center whitespace-normal break-words">
                 {move || tournament().name}
             </h1>
-            <div class="w-full text-center whitespace-normal break-words">
-                {move || tournament().description}
-            </div>
+            <div class="flex flex-col py-2 prose dark:prose-invert" inner_html=markdown_desc />
```

Comment:
To stop single line long descriptions from breaking the layout of the page the classes need to change a bit, something like this works, as also noted for the create page
```html
            <div class="p-4 w-full break-words prose dark:prose-invert" inner_html=markdown_desc />
```